logsfrom/logsfrom.py

```diff
@@ -94,9 +94,9 @@ async def logsfrom(
             channel = ctx.channel
             ctxc = ctx
         if not channel.permissions_for(ctx.me).read_message_history:
-            raise commands.BotMissingPermissions(["read_message_history"])
+            raise commands.BotMissingPermissions(discord.Permissions(read_message_history=True))
         if not await check_permissions(ctxc, {"read_message_history": True}):
-            raise commands.MissingPermissions(["read_message_history"])
+            raise commands.MissingPermissions(discord.Permissions(read_message_history=True))
```

Comment:
```suggestion
            raise commands.MissingPermissions(["read_message_history"])
```
[Red doesn't override `MissingPermissions`.](https://github.com/Cog-Creators/Red-DiscordBot/blob/1fa76bf43f0df9eecf264c0f21dd3d3505d89d60/redbot/core/commands/errors.py#L6)